### PR TITLE
Allow action to work without using built in subscription

### PR DESCRIPTION
**Pull request by TheMonDon**
Tested and works the same.

No harm in adding this and useful for anyone who may be joining the voice channel in a different way... say with a mod :P

### CODE REVIEW

**Comment by TheMonDon** [comment]
The reason I would like this implemented is for the play music mod: https://github.com/TheMonDon/mods/blob/DBM-2.2.0/actions/play_music_MOD.js#L76

File: actions/check_is_bot_in_voice_channel.js [hunk 1]
```diff
@@ -100,7 +100,13 @@ module.exports = {
 	action(cache) {
 		const data = cache.actions[cache.index];
 		const Audio = this.getDBM().Audio;
-		const result = cache.server ? Audio.getSubscription(cache.server) : false;
+		let result = false;
+		if (cache.server) {
+			result = Audio.getSubscription(cache.server);
+			if (!result) {
+				result = !!cache.server.members?.me?.voice?.channel;
```
Comment (almeidx) [inline actions/check_is_bot_in_voice_channel.js:107]: No reason to invoke the `channel` getter

```suggestion
				result = !!cache.server.members?.me?.voice?.channelId;
```

File: actions/check_is_bot_in_voice_channel.js [hunk 2]
```diff
@@ -100,7 +100,13 @@ module.exports = {
 	action(cache) {
 		const data = cache.actions[cache.index];
 		const Audio = this.getDBM().Audio;
-		const result = cache.server ? Audio.getSubscription(cache.server) : false;
+		let result = false;
+		if (cache.server) {
+			result = Audio.getSubscription(cache.server);
```
Comment (almeidx) [inline actions/check_is_bot_in_voice_channel.js:105]: Isn't this action supposed to store a boolean value?

```suggestion
			result = !!Audio.getSubscription(cache.server);
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/check_is_bot_in_voice_channel.js (hunk 2 above)
Comment (TheMonDon) [reply actions/check_is_bot_in_voice_channel.js:105]: Yes it should, I didn't think about that and kept it the same as Robert had it.

**Pull request merged**